recipes/plasmidid/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/config_files
+mkdir -p $PREFIX/config_files
+
+cp bin/* $PREFIX/bin
+cp plasmidID $PREFIX/bin
+
+cp config_files/* $PREFIX/bin/config_files
+
+echo "INSTALING MASH 2"
+echo "#############################################"
+
+git clone https://github.com/marbl/Mash.git
```

Comment:
This isn’t reproducible. You need to at least make sure you use a certain revision.

What was the problem with specifying mash as a normal dependency?

Reply:
Thank you for this comment. Now mash is installed from a reproducible source.
Indeed, mash is very problematic. By default conda installs v1.1 and version 2.2 can't be forced, it appears in conda search list but won't install.
If I add mash as a normal dependency basically all packages return a version conflict. This problem has to be addressed, but since the recipe seems fine, I don't know who should fix it.

Reply:
What happens if you specify `mash >= 2.2`?

Reply:
In the recipe `conda-build` thows an error like that:

```
Package curl conflicts for:
prokka -> blast[version='>=2.7.1'] -> curl[version='>=7.64.1,<8.0a0']
samtools -> curl[version='>=7.59.0,<8.0a0|>=7.64.0,<8.0a0|>=7.64.1,<8.0a0']
blast -> curl[version='>=7.64.1,<8.0a0']
bedtools -> curl[version='>=7.64.1,<8.0a0']
```

For every package and dependency, either with mash and mash >= 2

With` conda install `also fails to install, I have to specify the build (mash=2.2.2=h3d38be6_1) and add -c conda-forge to install it properly, but within the recipe I get the previous error for every package.

Reply:
Since you mention that you have to specify `-c conda-forge` when you ran `conda install`: Did you perhaps leave out `conda-forge` when you ran `conda build`? Bioconda requires conda-forge, so if this fails, it’s not a suprise. I suggest you [add both conda-forge and bioconda](https://bioconda.github.io/user/install.html#set-up-channels) in the correct order to your `.condarc`.

Reply:
I’ve just tested the recipe locally and when I add `mash` to the dependencies and remove the corresponding part from `build.sh`, it works fine.

I also ran `conda create -n tmp mash` just now and it works fine (with `conda-forge` and `bioconda` in the channel list). Note that when you install packages in Conda, it’s always recommended that you create a new environment to avoid potential conflicts with the packages already installed. These conflicts could explain the trouble you had installing it.